sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -304,6 +305,7 @@ case class CatalogTable(
     if (owner.nonEmpty) map.put("Owner", owner)
     map.put("Created", new Date(createTime).toString)
```

Comment:
Plan to backport this to the previous version. Thus, will not change it in this PR.